bower.json

```diff
@@ -7,12 +7,12 @@
     "consumerfinance.gov",
     "capital-framework"
   ],
-  "authors": [
-    {
-      "name": "Consumer Financial Protection Bureau",
-      "url": "https://cfpb.github.io/"
-    }
-  ],
+  "author":
+  {
+    "name": "Consumer Financial Protection Bureau",
+    "email": "tech@cfpb.gov",
+    "url": "https://cfpb.github.io/"
+  },
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/cd4543d6c4da636c8f562be5876cb7699caf84e2 Also updated `url` > `homepage` per the spec.